3ds Max/Max2Babylon/Exporter/BabylonExporter.IMaterialExporter.cs

```diff
@@ -81,7 +81,7 @@ public struct ClassIDWrapper : IEquatable<ClassIDWrapper>
         public static readonly ClassIDWrapper Switch_Shader_Material = new ClassIDWrapper(0x7e73161f, 0xa844c228);
         public static readonly ClassIDWrapper Two_Sided_Material = new ClassIDWrapper(0x7e73161f, 0x7ffd6281);
 
-
+        public static readonly ClassIDWrapper Editable_Poly = new ClassIDWrapper(469250957, 422535320);
```

Comment:
I'm not sure if this is the best place to put this class IDWrapper definition, but lets leave it here for now till we move all the magic numbers somewhere logical

Reply:
me too, in the future, we should make a refactoring of all the constants we added in the code, including exportingParameters, userDefinedProperties etc